### PR TITLE
created calendar

### PR DESCRIPTION
**Pull request by barantarasNEW**
https://barantarasNEW.github.io/layout_calendar/
https://barantarasNEW.github.io/layout_calendar/report/html_report/

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+}
+.calendar__day:nth-child(6)::before {
+  content: "6";
+}
+.calendar__day:nth-child(7)::before {
+  content: "7";
+}
+.calendar__day:nth-child(8)::before {
+  content: "8";
+}
```
Comment (liliia-livandovskaya) [inline css/main.css]: use `for` to set content

@for $day from 1 through $month-length {
      &:nth-child(#{$day})::before {
        content: "#{$day}";
      }
    }

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+.calendar__day:nth-child(30)::before {
+  content: "30";
+}
+.calendar__day:nth-child(31)::before {
+  content: "31";
+}
+.calendar__day:hover {
+  transform: translateY(-20px);
+  background-color: #ffbfcb;
+}
```
Comment (liliia-livandovskaya) [inline css/main.css]: don't forget to add empty line in the end of the file

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
GJ! Pay attention to the comments

File: css/main.css (hunk 2 above)
Comment (barantarasNEW) [reply css/main.css]: image
???

File: src/styles/utils/_extends.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+%displayN {
```
Comment (Viktor-Kost) [inline src/styles/utils/_extends.scss]: No needs to create placeholder with one line of rule

File: css/main.css (hunk 1 above)
Comment (Viktor-Kost) [reply css/main.css]: So, i see you have already used a loop for creating date, it means that you don't need these any more

.....
`.calendar__day:nth-child(7)::before {
  content: "7";
}
.calendar__day:nth-child(8)::before {
  content: "8";
}`
.....

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Nice! 
Pay attention to unresolved conversations.
If you need any help - ask at fe_chat

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
If we set 29 days in a modifier month-length calendar should show 29 days, but it does not work. Please fix it 

Screenshot 2022-12-09 at 11 41 25

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Hello!)

Why do you need both main.scss and main.css? 
You connect only main.scss to your index.html
So delete main.css file

**Comment by barantarasNEW** [comment]
sorry , forgot

**Review by Viktor-Kost — APPROVED** [review]
👍

**Pull request closed without merging**